src/value.h

```diff
@@ -163,9 +163,18 @@ bool SIValue_IsTrue(SIValue v);
 const char *SIType_ToString(SIType t);
 
 // Prints all individual types represented by 't', multiple types are separated by comma,
-// to a given buffer with length (bufferLen)
+// to a given buffer with length (bufferLen) last representation is preceeding by 'or' when more than 1 type
 void SIType_ToMultipleTypeString(SIType t, char *buf, size_t bufferLen);
 
+// Prints all individual types represented by 't', multiple types are separated by seperator,
+// to a given buffer with length (bufferLen), do not treat the last type differently
+// Return the number of bytes written in this call
+size_t SIType_ToMultipleTypeStringSimple(SIType t, char seperator, char *buf, size_t bufferLen);
```

Comment:
Please consolidate these two function, I think we can just add `char separator ` to `SIType_ToMultipleTypeString`

Reply:
SIType_ToMultipleTypeString return human readable format like: 
Null, TimeDate or String 
All params are seperated by ',' and the last one has 'or'

SIType_ToMultipleTypeStringSimple is intend for machine parsing and the format is:
Null|TimeDate|String